Вторая и третья часть лабораторной работы с Library выполнена

PopovMatvey opened the pull request:
Был свёрстан основной контент в соответствии с макетом:
1. Боковая панель и её элементы;
2. Список книг
3. Фильтры и поиск. 
Была применена семантическая вёрстка и имена классов соответствуют описанным блокам.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Вторая часть лаб. работы с Library выполнена

Был свёрстан основной контент в соответствии с макетом:
1. Боковая панель и её элементы;
2. Список книг
3. Фильтры и поиск. 
Была применена семантическая вёрстка и имена классов соответствуют описанным блокам.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

djilua1 commented:
![image](https://user-images.githubusercontent.com/43634050/166515787-2febd24d-ece0-40ce-acd0-c2ed2d6e244c.png)
Сделай нормальное поведение для маленького разрешения, просто флексами перенос книг при сужении (flex-wrap).
Давай книги сразу через js добавлять, так не очень смотрится, слишком громоздко.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -94,14 +95,368 @@ svg {
+}
+
+.main-content {
+    width: 84.5%;
```
Не дроби проценты.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -94,14 +95,368 @@ svg {
+    color: rgb(255, 255, 255);
+    text-transform: uppercase;
+    text-align: center;
+    z-index: 38;
```
Не стоит ставить рандомные значения для z-index.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -94,14 +95,368 @@ svg {
+    height: 30px;
+    border: none;
+    outline: none;
+    border-radius: 15.5px;
```
Пиксель минимальная единица измерения, его тоже не нужно дробить.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -94,14 +95,368 @@ svg {
+    display: block;
+    font-size: 12px;
+    color: #6f7d95;
+    margin: 11px 0px 12px 0px;
```
Разбей на несколько файлов, слишком громоздко выглядит. Отдельно хэдер, футер, сайдбар и мейн часть.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create script.js

PopovMatvey committed:
Исправления по второй части лаб. работы Library

Были исправлены все шесть замечаний и выполнены все линтеры.
1. Была исправна ошибка с адаптивом основной части, был добавлен flex-wrap для переноса книг при сужении;
2. Все книги теперь добавляются при помощи js;
3. и 5. дробные проценты и пиксели теперь отсутствуют;
4. Сгенерированные Photoshop z-index были удалены;
6. css теперь разбит на несколько ключевых файлов: header.css , footer.css , slide-panel.css , main-content.css и style.css для общих стилей, не привязанных к определённому блоку.

PopovMatvey committed:
Добавлено больше адаптива

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Третья часть лабораторной работы Libracy выполнена

Был реализован следующий функционал:
1. Добавление новой книги с использованием модального окна;
2. Был реализован поиск по названию книги и по автору;
3. Есть возможность изменить рейтинг при нажатии на название книги
4. Был реализован фильтр "Most Popular".

PopovMatvey commented:
По лабораторной работе с событиями был реализован следующий функционал:
1. Добавление новой книги с использованием модального окна;
2. Был реализован поиск по названию книги и по автору;
3. Есть возможность изменить рейтинг при нажатии на название книги
4. Был реализован фильтр "Most Popular".
